eZ/Publish/Core/Search/Common/IncrementalIndexer.php

```diff
@@ -74,6 +74,7 @@
      * @param int[] $contentIds
      * @param bool $commit
      * @param bool $continueOnError
+     * @return array
```

Comment:
`@return int[] Returns id's content items we could not index as field type indexer threw a InvalidIndexDataException, details on the exception can be found in log message.`